### ISSUE TITLE
paper 中的所有模型的MSE/MAE结果是否就是直接用的仓库中设定的参数

### ISSUE THREAD

**Issue by f0rzekeN** [Ladbaby/PyOmniTS#3]
比如您的模型是否就用的如下仓库的 .sh 得出的结果
use_multi_gpu=0
if [ $use_multi_gpu -eq 0 ]; then
    launch_command="python"
else
    launch_command="accelerate launch"
fi

model_name="$(basename "$(dirname "$(readlink -f "$0")")")" # folder name

dataset_root_path=storage/datasets/HumanActivity
model_id_name=$model_name
dataset_name=$(basename "$0" .sh) # file name

seq_len=$((3000))
for pred_len in 300; do
    for batch_size in 32; do
        $launch_command main.py \
            --is_training 1 \
            --loss "MSE" \
            --d_model 128 \
            --n_layers 2 \
            --n_heads 4 \
            --use_multi_gpu $use_multi_gpu \
            --dataset_root_path $dataset_root_path \
            --model_id $model_id_name'_'$seq_len'_'$pred_len \
            --model_name $model_name \
            --dataset_name $dataset_name \
            --features M \
            --seq_len $seq_len \
            --pred_len $pred_len \
            --enc_in 12 \
            --dec_in 12 \
            --c_out 12 \
            --train_epochs 300 \
            --patience 10 \
            --val_interval 1 \
            --itr 5 \
            --batch_size $batch_size \
            --learning_rate 0.001
    done
done

**Comment by Ladbaby** [comment]
论文中包含很多不同的回望/预测窗口设定的实验，而仓库中只提供了模型相对鲁棒的一组参数。

**Comment by daybraeklaxry** [comment]
两张大表是否是仓库里的参数？谢谢作者

**Comment by f0rzekeN** [comment]
只考虑 MSE/MAE 两张大表的结果，是否是仓库中的参数

**Comment by Ladbaby** [comment]
是的
